ui/packages/image/src/SketchSettings.svelte

```diff
@@ -36,9 +41,14 @@
 
 	{#if mode !== "mask"}
 		<span class="absolute top-6 right-0">
-			<IconButton Icon={Color} on:click={() => (show_col = !show_col)} />
+			<IconButton
+				Icon={Color}
+				aria-label="Select brush color"
+				on:click={() => (show_col = !show_col)}
+			/>
 			{#if show_col}
 				<input
+					aria-label="Brush color"
```

Comment:
`label`

Reply:
I think this comment's on the wrong line, the <input> will need to keep aria-label but I updated the <IconButton> prop line.